agent/lm_agent/utils.py

```diff
@@ -17,6 +18,10 @@ async def run_command(command_line: str):
     output = str(stdout, encoding=ENCODING)
 
     if proc.returncode != 0:
-        logger.error(f"Error: {output} | Return Code: {proc.returncode}")
-        return None
+        logger.error(
+            f"Command {command_line} failed!",
+            f"Error: {output}",
+            f"Return code: {proc.returncode}",
+        )
```

Comment:
I just tested this and I couldn't get the three messages printed. As far as I know, only the first argument is the message to be printed, right? [Reference](https://docs.python.org/3.10/library/logging.html#logging.error).

Also, @dusktreader once recommended that this kind of log should be done only in one line instead of multiple. Do you mind changing it?

Reply:
Yes, the reason is that you want log messages that are part of a single message to appear as one entry in log aggregators and such. In this case, I think you could just join each part with a newline.

Reply:
You're right, I fixed it in #189.